core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -676,7 +676,7 @@ private[ui] class TaskDataSource(
 
   private var _tasksToShow: Seq[TaskData] = null
 
-  override def dataSize: Int = stage.numCompleteTasks + stage.numFailedTasks + stage.numKilledTasks
```

Comment:
@vanzin I think you made this change, why not also track the running tasks? I'm not sure what is your original purpose.

Reply:
Probably just a side effect of me testing mostly with replayed logs and short-lived tasks.